specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/networkInterface.json

```diff
@@ -897,10 +897,34 @@
         "provisioningState": {
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the network interface IP configuration."
+        },
+        "privateLinkConnectionProperties": {
```

Comment:
The "properties" suffix seems unnecessary.  "publicIPAddress" and "subnet" aren't using the suffix and its not typical across other resources.

Reply:
Hi Pilor, 
So regarding to your question,  the property's name is '**privateLinkConnectionProperties**' from the Networking RP server side, it's been public and we cannot change it. So I think we should keep the name as is. Thank you.

Reply:
Public how?  If this is already public then what am I reviewing here?

Reply:
Hi Pilor,
My meaning is the NRP side has already released the REST api for this part. And the property is already there for a while which named 'privateLinkConnectionProperties'. So I don't think there is a reason to change the name.
I attached one of example of return json string for network interface for your reference:
```
{
  "name": "PEVnet1.nic.b9414b57-e682-40b9-ada8-312e40e2c88b",
  "id":
"/subscriptions/15aa9ba4-9bd3-499f-ba12-9a5736d0cd68/resourceGroups/PrivateEndpointRunnerTestToStaticRGRGWestcentralusSLBV2Dev/providers/Microsoft.Network/networkInterfaces/PEVnet1.nic.b9414b57-e682-40b9-ada8-312e40e2c88b",
  "etag": "W/\"b2dd33b7-92d1-460a-baa6-d5f55960e4aa\"",
  "properties": {
    "provisioningState": "Succeeded",
    "resourceGuid": "7e3a996f-09a0-41e2-9f6d-0841110f7823",
    "ipConfigurations": [
      {
        "name": "sqlServer-sqlServer.privateEndpoint",
        "id":
"/subscriptions/15aa9ba4-9bd3-499f-ba12-9a5736d0cd68/resourceGroups/PrivateEndpointRunnerTestToStaticRGRGWestcentralusSLBV2Dev/providers/Microsoft.Network/networkInterfaces/PEVnet1.nic.b9414b57-e682-40b9-ada8-312e40e2c88b/ipC
onfigurations/sqlServer-sqlServer.privateEndpoint",
        "etag": "W/\"b2dd33b7-92d1-460a-baa6-d5f55960e4aa\"",
        "type": "Microsoft.Network/networkInterfaces/ipConfigurations",
        "properties": {
          "provisioningState": "Succeeded",
          "privateIPAddress": "10.0.0.4",
          "privateIPAllocationMethod": "Dynamic",
          "subnet": {
            "id":
"/subscriptions/15aa9ba4-9bd3-499f-ba12-9a5736d0cd68/resourceGroups/PrivateEndpointRunnerTestToStaticRGRGWestcentralusSLBV2Dev/providers/Microsoft.Network/virtualNetworks/vNet1-nr3kriysgtcxk/subnets/subnet1"
          },
          "primary": true,
          "privateIPAddressVersion": "IPv4",
          "privateLinkConnectionProperties": {
            "groupId": "sqlServer",
            "requiredMemberName": "sqlServer",
            "fqdns": [
              "sql-for-test-4nuppzq4x37iq.database.windows.net"
            ]
          }
        }
      }
    ],
    "dnsSettings": {
      "dnsServers": [],
      "appliedDnsServers": [],
      "internalDomainNameSuffix": "vji40krafe4ujcvrvvzyhdrltc.yx.internal.cloudapp.net"
    },
    "macAddress": "",
    "enableAcceleratedNetworking": false,
    "enableIPForwarding": false,
    "hostedWorkloads": [],
    "tapConfigurations": [],
    "privateEndpoint": {
      "id": "/subscriptions/15aa9ba4-9bd3-499f-ba12-9a5736d0cd68/resourceGroups/PrivateEndpointRunnerTestToStaticRGRGWestcentralusSLBV2Dev/providers/Microsoft.Network/privateEndpoints/PEVnet1"
    }
  },
  "type": "Microsoft.Network/networkInterfaces"
}
```